plugins/cue_points/quiz/services/QuizService.php

```diff
@@ -168,11 +168,13 @@ public function serveAction($entryId, $quizFileType)
 	 *
 	 * @action getUrl
 	 * @param string $entryId
+	 * @param KalturaQuizFileType $quizFileType
 	 * @return string
 	 * @throws KalturaErrors::ENTRY_ID_NOT_FOUND
 	 * @throws KalturaQuizErrors::PROVIDED_ENTRY_IS_NOT_A_QUIZ
+	 * @throws KalturaQuizErrors::NO_SUCH_FILE_TYPE
```

Comment:
It's already verified by the request deserializer, no need to check it yourself.